regulations/templates/regulations/node.html

```diff
@@ -13,6 +18,9 @@
     {{node.marked_up|safe}}
 {% endif %}
 </p>
+{% if root.node_type == "preamble" and node.node_type == "regtext" %}
+  <div class="activate-write" data-section="{{ node.markup_id }}">Comment on paragraph in section {{ node.label_id }}</div>
+{% endif %}
```

Comment:
Hrm. I don't think we should be having `regtext` nodes inside `preambles` -- I think this guard will need to change in the future. Maybe just `not node.title`